people.md

```diff
@@ -0,0 +1,29 @@
+# pandas People
+
+## Benevolent Dictator for Life
+
+Wes McKinney is the Benevolent Dictator for Life (BDFL).
+
+## Project Core Team
+
+- Tom Augspurger
+- Phillip Cloud
+- Andy Hayden
+- Stephan Hoyer
+- Thomas Kluyver
+- Wes McKinney
+- Wouter Overmeire
+- Jeff Reback
+- Skipper Seabold
+- Chang She
+- Sinhrks
+- Jeff Tratner
+- Joris Van den Bossche
+
+## Project NumFOCUS Board
```

Comment:
Thanks -- I've incorporated and appopriately edited the Jupyter bits about this, and indicated that the project NumFOCUS Board and Subcommittee are the same group of people. This is important to avoid future conflicts of interest around the direction of project funds through NumFOCUS